package collision_map_creator_msgs.msgs;
import "vec2d.proto";

message CollisionMapRequest
{
  required Vec2d upperLeft  = 1;
  required Vec2d upperRight = 2;
  required Vec2d lowerRight = 3;
  required Vec2d lowerLeft  = 4;
  required double     height     = 5;
  required double     resolution = 6;
  optional string     filename   = 7 [default = ""];
  optional int32      threshold  = 8 [default = 255];
}
